include/graphics/shaderloader.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <GL/glew.h>
```

Comment:
Are these really needed in this file? Try to keep includes down to the places they're actually used.